src/EnergyPlus/DataSurfaces.hh

```diff
@@ -215,6 +215,15 @@ namespace DataSurfaces {
         "OnIfHighZoneAirTempAndHighSolarOnWindow",
         "OnIfHighZoneAirTempAndHighHorizontalSolar"};
 
+    enum RefAirTemp // Parameters to indicate reference air temperatures for inside surface temperature calculations
+    {
+        Invalid = -1,
```

Comment:
No need to have AirTemp in both the `enum` type name and the `enum` value names.